flex/http.py

```diff
@@ -120,10 +126,27 @@ def _normalize_python3_urllib_request(request):
     )
 
 
+def _normalize_tornado_request(request):
```

Comment:
(arbitrary thread anchor)

Are you opposed to adding tornado as a dependency for tests? I could mock their interface, but I generally prefer to use the real thing when possible.